### PR TITLE
Enable verbose credential obtainment logging on debug

### PR DESCRIPTION
**Pull request by worzel666**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Reference: #25 

When DebugLogging is enabled, toggle on the verbose credentials errors.

### CODE REVIEW

**Review by ewbankkit — APPROVED** [review]
LGTM :rocket:.

**Comment by ewbankkit** [comment]
@gdavison Can you take a look please?

File: awsauth.go [hunk 1]
```diff
@@ -261,6 +261,7 @@ func GetCredentials(c *Config) (*awsCredentials.Credentials, error) {
 	}
 
 	if c.DebugLogging {
+                awsConfig.CredentialsChainVerboseErrors = aws.Bool(true)
```
Comment (ewbankkit) [inline awsauth.go:264]: Looks like a formatting issue.

```suggestion
		awsConfig.CredentialsChainVerboseErrors = aws.Bool(true)
```

Comment (worzel666) [reply awsauth.go]: My bad! Fixed.

**Comment by gdavison** [comment]
Hi @worzel666, thanks for the PR. We've merged #63, so that verbose logging is always enabled. This will give a more useful error without needing to re-run Terraform with debug logging enabled.
I didn't see your PR before writing mine.

**Comment by dcu** [comment]
was this change reversed?

**Pull request closed without merging**